### DISCUSSION TITLE
Using App Directory & remove Pages directory in Next.js 13

### DISCUSSION THREAD

**Discussion by JGSolutions** [vercel/next.js#42897] [Help]
Not clear about this. But I am trying to implement the new app directory in version 13 and want to export as static pages but it complains when i run 'next build && next export'

error:
PageNotFoundError: Cannot find module for page: /

What is the proper way to export static pages using hte new App Directory?

**Comment by wwwdepot** [comment]
`next export` is in development for app directory, see roadmap https://beta.nextjs.org/docs/app-directory-roadmap

**Reply by JGSolutions** [reply]
thanks for reply. Didn't know was expecting it to work.
